_data/other.yml

```diff
@@ -103,6 +103,13 @@ websites:
       img: commuterbenefits.png
       tfa: No
       twitter: CommuterBenefit
+      
```

Comment:
Ahh.  Sorry about that.  Stupid Notepad++.  Should have used vim.  ;)

Pushing that and the updated icon (I'll use the favicon).  Thanks for those!